GetStream/stream-chat-swift
# [CIS-1189] Fix long file names overlapped by the close button

### 🔗 Issue Link
CIS-1189

### 🎯 Goal
Fix long file names being overlapped by the close button "X".

### 🛠 Implementation
Add a hardcoded margin-right to the file name label. Currently, the "X" button is part of a parent view, so we can't make a constraint to that button.

### 📝 Changes
- File long file names being overlapped by the close button
- Make sure to truncate files at the middle, so that the extension of the file is always visible

### 🎨 UI Changes

| Before | After |
| --- | --- |
| <img width="331" alt="image" src="https://user-images.githubusercontent.com/12814114/160428624-9c9e21e9-d47b-4497-8eae-f59fa46a444c.png"> | <img width="334" alt="image" src="https://user-images.githubusercontent.com/12814114/160428003-56cdb286-527a-4049-9606-62a5444256dc.png"> |

### 🧪 Testing
- Open a channel
- Import a file with a long name
- Verify the file is truncated at the middle and doesn't go below the close button

### ☑️ Contributor Checklist
